coalib/bearlib/spacing/SpacingHelper.py

```diff
@@ -25,19 +25,12 @@ def get_indentation(self, line: str):
         :param line: A string to check for indentation.
         :return:     The indentation count in spaces.
         """
-        count = 0
```

Comment:
I believe all of this code is needed here. Variable count should check for white spaces as well as tabs.

Reply:
If the line has the indentation with spaces, this part ```len(line) - len(line.lstrip())``` can count. But the problem is if we have tabs or worst (i don't know.. in some case) if we have tabs and spaces. Let's see some examples:
### With Spaces
```python
>>> another_line = "    count += 1"
>>> len(another_line) - len(another_line.lstrip())
4
```
### With Tabs
```python
>>> another_line = "\tdef function():"
>>> len(another_line) - len(another_line.lstrip())
1
>>> len(another_line) - len(another_line.lstrip('\t'))
1
```

### With Tabs and Spaces
```python
>>> another_line = " \tif everything_is_okay(some_value):"
>>> len(another_line) - len(another_line.lstrip())
2
>>> len(another_line) - len(another_line.lstrip('\t'))
0
```

The command counts once in each tab, but the method get_indentation() has to return the indentation count in **spaces**. Currently, this method searchs char by char (using a for loop) and (if this char is a white space) adds one to the count or (if this char is a tab) makes a calculation based on the tab size and it adds the result to the count.

To fix this problem with the tabs i found out the **expandtabs**. This command, *roughly speaking*, replaces tabs with spaces:

### With Tabs and expandtabs
```python
>>> another_line = "\tdef function():"

>>> another_line.expandtabs(4)
'    def function():'

>>> line_expanded = another_line.expandtabs(4)

>>> len(line_expanded) - len(line_expanded.lstrip())
4
```
### With Tabs, Spaces and expandtabs:
```python
>>> another_line = " \tif everything_is_okay(some_value):" 

>>> another_line.expandtabs(4)
'    if everything_is_okay(some_value):' # Having a space before the tab shouldn't make any difference

>>> line_expanded = another_line.expandtabs(4)

>>> len(line_expanded) - len(line_expanded.lstrip())
4
```

PS: The "4" is the tab size.

Reply:
What if no tabs are present and just spaces. Does it return an error in that case ?

Reply:
Doesn't, expandtabs will be execute but the line will not change because doesn't has tabs to replace (just spaces). After that the count will occur :tropical_drink:

Reply:
Okay. This looks convincing to me. But your build is failing on circle ci. Did you run `pytest` locally?

Reply:
I did, but I really do not know why the build failed. 
This builded with no problem in the last 2 updates (that i only changed the commit description) :S

Reply:
That is not possible. Most probably there is something that has been changed in your code while rebuilding your PR.